variables.tf

```diff
@@ -154,7 +154,7 @@ variable "executor_firecracker_disk_space" {
 
 variable "executor_min_replicas" {
   type        = number
-  default     = 1
+  default     = 0
```

Comment:
should we keep 1? In case autoscaling doesn't work/isn't set up, it wouldn't start one automatically, I think.

Reply:
Good point.